recipes/fenics/build.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+
+if [[ "$(uname)" == "Darwin" ]]; then
+  export MACOSX_DEPLOYMENT_TARGET=10.9
+  export CXXFLAGS="-std=c++11 -stdlib=libc++ $CXXFLAGS"
+fi
+
+for pkg in dijitso ufl instant fiat ffc; do
+    echo "installing ${pkg}-${PKG_VERSION}"
+    git clone -q --depth 1 -b ${pkg}-${PKG_VERSION} https://bitbucket.org/fenics-project/${pkg}.git
+    pushd $pkg
+    pip install --no-deps .
+    popd
+done
+
+pkg=dolfin
+echo "installing ${pkg}-${PKG_VERSION}"
+git clone -q --depth 1 -b ${pkg}-${PKG_VERSION} https://bitbucket.org/fenics-project/${pkg}.git
+pushd $pkg
+# apply patches
+git apply "${RECIPE_DIR}/swig-py3.patch"
+if [[ "$(uname)" == "Darwin" ]]; then
+    git apply "${RECIPE_DIR}/clang6-explicit-in-copy.patch"
```

Comment:
Same [comment](https://github.com/conda-forge/staged-recipes/pull/1785/files#r84674471) for the patch.